apps/modernization-ui/src/apps/page-builder/components/AddValueset/AddValueset.tsx

```diff
@@ -38,6 +40,19 @@ export const AddValueset = ({ modalRef, updateCallback }: Props) => {
             setSearchValueSet?.({ valueSetNm: body.valueSetNm, codeSetGroupId: body.codeSetGroupId });
         });
     };
+    const handleUpdateValueSet = () => {
+        const update = {
+            valueSetNm: name,
+            valueSetCode: code,
+            codeSetDescTxt: desc
+        };
+        ValueSetControllerService.updateValueSetUsingPost({
+            authorization,
```

Comment:
Suggestion: Could use authorization(), instead of getAuthorization()

Reply:
getAuthorization is just rename/instance of authorization
Line 6` import { authorization as getAuthorization } from 'authorization';`